### DISCUSSION TITLE
How to switch between MobileFormatter vs BinaryFormatter

### DISCUSSION THREAD

**Discussion by EricNgo1972** [MarimerLLC/csla#1811] [Questions] [answered]
Hello, 

What is the proper way to config SerializationFormatter in code ?

The Csla4 book, which used the app.config or web.config seems to not suitable for modern .net app anymore.

Thank you.

**Comment by rockfordlhotka** [comment, marked as answer]
Something like this:

```c#
      app.UseCsla(config => config.Configure()
        .Serialization()
          .SerializationFormatter("BinaryFormatter")
      );
```
